src/kube-runtime/build/kube-runtime.dockerfile

```
@@ -20,11 +20,11 @@ FROM python:2.7-alpine3.8
 
 ARG BARRIER_DIR=/opt/frameworkcontroller/frameworkbarrier
 
-WORKDIR /usr/local/pai
+RUN mkdir -p /kube-runtime/src
+WORKDIR /kube-runtime/src
 
 COPY src/ ./
 COPY --from=frameworkcontroller/frameworkbarrier:v0.3.0 $BARRIER_DIR/frameworkbarrier ./init.d
-RUN mkdir -p ./logs && \
-    chmod -R +x ./
+RUN chmod -R +x ./
 
-CMD ["/bin/sh", "-c", "/usr/local/pai/init"]
+CMD ["/bin/sh", "-c", "/kube-runtime/src/init"]
```

You can move the sources in dockerfile

```suggestion
CMD ["/bin/sh", "-c", "mv ./* /usr/local/pai && /usr/local/pai/init"]
```